auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -190,13 +343,41 @@
                   <span class="open-action">
                     <v-btn
                       color="primary"
-                      :class="['open-action-btn', getButtonLabel(item.status).toLowerCase()]"
+                      :class="['open-action-btn', { active: item.nrNumber && !['APPROVED', 'REJECTED', 'ABANDONED'].includes(item.status) }]"
                       :data-test="getIndexedTag('view-continuation-button', item.id)"
                       @click="view(item.id)"
                     >
                       {{ getButtonLabel(item.status) }}
                     </v-btn>
                   </span>
+                  <!-- More Actions Menu -->
+                  <span
+                    v-if="item.nrNumber && !['APPROVED', 'REJECTED', 'ABANDONED'].includes(item.status)"
+                  >
+                    <v-menu
+                      v-model="dropdown[item.nrNumber]"
+                      offset-y
+                      nudge-left="158"
+                    >
+                      <template #activator="{ on }">
+                        <v-btn
+                          color="primary"
+                          class="more-actions-btn"
+                          v-on="on"
+                        >
+                          <v-icon>{{ dropdown[item.nrNumber] ? 'mdi-menu-up' : 'mdi-menu-down' }}</v-icon>
+                        </v-btn>
+                      </template>
+                      <v-list>
+                        <v-list-item @click="goToNameXUI(item.nrNumber)">
+                          <v-list-item-subtitle>
+                            <v-icon style="font-size: 14px">mdi-format-list-bulleted</v-icon>
+                            <span class="pl-2">Open Name Request</span>
+                          </v-list-item-subtitle>
+                        </v-list-item>
+                      </v-list>
+                    </v-menu>
+                  </span>
```

Comment:
I found a bug: the sub-menu opens multiple times when I click the down arrow to open it. Double-check this code, or maybe it needs a 'key' prop?.

![image](https://github.com/user-attachments/assets/e4b501f3-4e8f-4020-83cb-6b081b102fac)

Reply:
Oh, it seems two reviews have the same NR number, and maybe that's why it's happening.

I don't know if duplicate NRs are valid (eg, it could be for a rejected app and then for a valid app), so if possible let's support this scenario for now.

Reply:
This is now fixed. Resolving.